### PR TITLE
Adds code coverage report via Coveralls

### PR DESCRIPTION
**Pull request by martyn-w**
NB: requires setting env variable `COVERALLS_REPO_TOKEN` in codeship

### CODE REVIEW

**Comment by martyn-w** [comment]
See code coverage in Coveralls here: https://coveralls.io/github/antleaf/nims-hyrax?branch=develop

File: README.md [hunk 1]
```diff
@@ -221,3 +221,9 @@ There is [docker documentation](https://docs.docker.com/storage/volumes/#backup-
 * As mentioned above, there is a `.env` file containing application secrets. This **must not** be checked into version control!
 * The system is configured on start-up using the `docker-entrypoint.sh` script, which configures users in the `seed/setup.json` file.
 * Importers are run manually in the container using the rails console. See [The project wiki](https://github.com/antleaf/nims-hyrax/wiki) for more information.
+
+## Code Status
+
+[![Codeship Status for antleaf/nims-hyrax](https://app.codeship.com/projects/d4cc8560-e430-0136-fffd-6a7889452552/status?branch=master)](https://app.codeship.com/projects/319029)
```
Comment (Steven-Eardley) [inline README.md]: develop branch might be more relevant here, since we're actively developing there.

Comment (martyn-w) [reply README.md]: @Steven-Eardley good catch, now tweaked

**Comment by Steven-Eardley** [comment]
one of the builds passed, at least

**Pull request merged**